examples/control-elements/media-chrome-menu-button.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta name="viewport" content="width=device-width">
+  <title>Media Chrome Captions Usage Examples</title>
+  <script type="module" src="../../dist/index.js"></script>
+  <script type="module" src="../../../dist/media-chrome-listbox.js"></script>
+  <script type="module" src="../../../dist/media-chrome-listitem.js"></script>
+  <script type="module" src="../../../dist/media-captions-listbox.js"></script>
+  <script type="module" src="../../../dist/media-chrome-menu-button.js"></script>
+  <style>
+    media-captions-listbox {
+      display: block;
+      height: fit-content;
+    }
+
+    media-chrome-listitem {
+      position: relative;
+      line-height: 1.5em;
+    }
+    media-chrome-listitem[aria-selected="true"]::before {
+      content: '✅';
+      position: absolute;
+      margin-left: -1.5em;
+    }
+
+    .examples {
+      margin-top: 20px;
+    }
+  </style>
+</head>
+<body>
+  <main>
+    <h1>Media Chrome Standard Video Usage Example</h1>
+    <media-chrome-menu-button>
+      <media-chrome-listitem>lorem</media-chrome-listitem>
+      <media-chrome-listitem>ipsum</media-chrome-listitem>
+      <media-chrome-listitem>dolor</media-chrome-listitem>
+      <media-chrome-listitem>sit</media-chrome-listitem>
+    </media-chrome-menu-button>
+
+    <br>
+
+    <media-chrome-menu-button aria-label="captions menu button">
+      <svg slot="button-content" aria-hidden="true" viewBox="0 0 26 24">
+        <path d="M22.83 5.68a2.58 2.58 0 0 0-2.3-2.5c-3.62-.24-11.44-.24-15.06 0a2.58 2.58 0 0 0-2.3 2.5c-.23 4.21-.23 8.43 0 12.64a2.58 2.58 0 0 0 2.3 2.5c3.62.24 11.44.24 15.06 0a2.58 2.58 0 0 0 2.3-2.5c.23-4.21.23-8.43 0-12.64Zm-11.39 9.45a3.07 3.07 0 0 1-1.91.57 3.06 3.06 0 0 1-2.34-1 3.75 3.75 0 0 1-.92-2.67 3.92 3.92 0 0 1 .92-2.77 3.18 3.18 0 0 1 2.43-1 2.94 2.94 0 0 1 2.13.78c.364.359.62.813.74 1.31l-1.43.35a1.49 1.49 0 0 0-1.51-1.17 1.61 1.61 0 0 0-1.29.58 2.79 2.79 0 0 0-.5 1.89 3 3 0 0 0 .49 1.93 1.61 1.61 0 0 0 1.27.58 1.48 1.48 0 0 0 1-.37 2.1 2.1 0 0 0 .59-1.14l1.4.44a3.23 3.23 0 0 1-1.07 1.69Zm7.22 0a3.07 3.07 0 0 1-1.91.57 3.06 3.06 0 0 1-2.34-1 3.75 3.75 0 0 1-.92-2.67 3.88 3.88 0 0 1 .93-2.77 3.14 3.14 0 0 1 2.42-1 3 3 0 0 1 2.16.82 2.8 2.8 0 0 1 .73 1.31l-1.43.35a1.49 1.49 0 0 0-1.51-1.21 1.61 1.61 0 0 0-1.29.58A2.79 2.79 0 0 0 15 12a3 3 0 0 0 .49 1.93 1.61 1.61 0 0 0 1.27.58 1.44 1.44 0 0 0 1-.37 2.1 2.1 0 0 0 .6-1.15l1.4.44a3.17 3.17 0 0 1-1.1 1.7Z"/>
+      </svg>
+      <media-captions-listbox slot="listbox" media-controller="mc"></media-captions-listbox>
```

Comment:
Does the slot _have_ to be a listbox? Or should we call this slot `menu` (since it's the menu button) and allow that slot to be more flexible?

Reply:
Called it `listbox` to match [selectmenu's slot](https://open-ui.org/prototypes/selectmenu). Ultimately, anything with a matching API would work.